uwp/Pdf-Viewer/Concepts-and-Features/Working-with-custom-stamp-annotation.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: Custom stamp annotation in UWP PDF Viewer control | Syncfusion
+description: Learn here all about Custom stamp annotation support in Syncfusion UWP PDF Viewer (SfPdfViewer) control and more.
+platform: UWP
+control: SfPdfViewerControl
+documentation: ug
+---
+
+# Working with custom stamp annotations in UWP PDF Viewer (SfPdfViewer)
+
```

Comment:
<img width="338" alt="image" src="https://github.com/syncfusion-content/uwp-docs/assets/65169077/bd4f861f-f436-4884-81dd-48b6420f8bad">

Convention of PDF Viewer is naming as either "PDF Viewer" or "PdfViewer". Please cross check

Reply:
Feedback addressed.